harvard-lts/CURIOSity
# Solr fix gems

**Fix Solr and update some gems**
* * *


# What does this Pull Request do?
Fixes the solr harvest that we discovered was broken, and bumps some gems for security

# How should this be tested?

Spin up this branch locally (or use dev, where it is deployed using trial). Test both mods and solr harvests (I use tedming for the solr harvest, demo for the mods) and confirm all basic functions work. VV will also test when this goes to QA

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
